packages/earljs/src/matchers/Error.ts

```diff
@@ -2,39 +2,34 @@ import { Newable } from '../types'
 import { smartEq } from '../validators/smartEq'
 import { Matcher } from './Base'
 
-/**
- * Matches any error with matching string
- */
 export class ErrorMatcher extends Matcher {
-  constructor(private readonly errorCls: Newable<Error> = Error, private readonly expectedMsg?: string) {
+  constructor(private readonly errorClass: Newable<Error> = Error, private readonly message?: string) {
     super()
   }
 
   check(v: unknown) {
-    if (!(v instanceof this.errorCls)) {
+    if (!(v instanceof this.errorClass)) {
       return false
     }
 
-    if (this.expectedMsg !== undefined) {
-      return smartEq(v.message, this.expectedMsg).result === 'success'
+    if (this.message !== undefined) {
+      return smartEq(v.message, this.message).result === 'success'
     }
 
     return true
   }
 
   toString() {
-    if (this.expectedMsg === undefined) {
-      return `${this.errorCls.name}`
+    if (this.message === undefined) {
+      return `${this.errorClass.name}`
     }
-    return `[${this.errorCls.name}: ${this.expectedMsg}]`
+    return `[${this.errorClass.name}: ${this.message}]`
   }
 
-  static make(expectedMsg: string): Error
-  static make(errorCls: Newable<Error>, expectedMsg?: string): Error
-  static make(errorClsOrExpectedMsg: string | Newable<Error>, expectedMsg?: string): Error {
-    if (typeof errorClsOrExpectedMsg === 'string' || errorClsOrExpectedMsg instanceof Matcher) {
-      return new ErrorMatcher(Error, errorClsOrExpectedMsg as any) as any
+  static make(classOrMessage: string | Newable<Error>, message?: string): Error {
+    if (typeof classOrMessage === 'string' || classOrMessage instanceof Matcher) {
+      return new ErrorMatcher(Error, classOrMessage as any) as any
     }
-    return new ErrorMatcher(errorClsOrExpectedMsg as any, expectedMsg) as any
```

Comment:
good catch - it is a leftover from a previous design in which this matcher was exposed to the users